bin/meli-test-account.js

```diff
@@ -0,0 +1,147 @@
+#!/usr/bin/env node
+require('dotenv').config({path: require('path').resolve(process.cwd(), '.env.test')})
```

Comment:
How is this CLI being used? shouldn't variable be passed as CLI args or flags instead of env variables? also this `dotenv` is only used when is on dev mode and we are always requiring it.

Reply:
I agree the answer to "which vars" is quite obscure ATM,.. I need to clarify which ones are being used. 
CLI args/flags I don't think would be too nice for this case, passing around values such as MONGODB_URL is not too comfortable, that's why I'm using dotenv. Maybe that could work for values more related to the script execution such as DEV_ACCOUNT_USERNAME and maybe PORT?

Also, using dotenv comes handy considering that a developer is already expected to have a .env setted up.